data_structures/bloom_filters/bloom_filter/bloom_filter.c

```diff
@@ -0,0 +1,167 @@
+/**
+ * @file bloom_filter.c
+ * @brief Implementation of a bloom filter
+ * https://en.wikipedia.org/wiki/Bloom_filter
```

Comment:
There should be a details section here, just copy some overview from Wikipedia should be good enough, but I'll leave the decision to you, Also check [this](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/data_structures/bloom_filter.cpp) out
```suggestion
 * @brief Implementation of a [bloom filter](https://en.wikipedia.org/wiki/Bloom_filter)
 * @details 
```